pymc3/variational/advi.py

```diff
@@ -182,7 +182,7 @@ def _calc_elbo(vars, model, n_mcsamples, random_seed):
 
     uw = tt.vector('uw')
```

Comment:
I already try to cast <code>uw</code> but no fixing. It just a bit issue with init test_value